org.openhab.binding.zigbee/README.md

```diff
@@ -43,16 +55,52 @@ Once a child is removed from the child table of a router, it will be asked to re
 
 Note that ZigBee compliant devices should rejoin the network seamlessly, however some non-compliant devices may not rejoin which may leave them unusable without a manual rejoin.
 
+**Values:** Timeout time in seconds. The table below lists the options that are shown in PaperUI and the equivalent values that can be set in a configuration file:
```

Comment:
I guess you could simply state that this is notionally defined in seconds rather than defining how many seconds are in each period?